README.md

```diff
@@ -28,10 +28,5 @@ gitignored so don't worry about accidentally adding log file to git repo. After
 getting file there's simple report generated with awk.
 
 ## Known issues
-- newrelic (and probably honeybadger) performs `HEAD /` requests to check if
-  site is up. This affects number of requests with `status=200` and `path="/"`.
-  It would be probably best to just filter those requests out from stats.
 - some requests are marked as failed by heroku (those with `code=XX`) and are
   filtered out right now. It's `~3000-4000` requests per day.
```

Comment:
I'd like some stats on those as well. Wether they are H14 out of memory or other :)